docker/README.md

```diff
@@ -65,9 +65,27 @@ Jenkins Kubernetes plugin slaves:
 
 -  [android-slave](./android-slave)
 
-## Build
+### Build
 
-To build slave execute docker build
+We are using s2i technology to for building mobile jenkins slaves. During the s2i build, a specific version of the Android SDK is installed and the user is asked to confirm all required licenses.
 
-    docker build . -t aerogear/jenkins-android-slave
+The android slave extends the openshift/jenkins-slave-base-centos7
+Any change in s2i dockerfile would require image build.
 
+    cd android-slave directory
+    docker build -t aerogear/jenkins-android-slave-s2i
+
+To include the android-sdk, download the required sdk, install and accept the license agreement
+The following script is an example of downloading the sdk installing and lastly a step to accept the license agreement
```

Comment:
replace `downloading and installing SDK`.  remove `step to accept the license agreement` - it's just part of installation.